install/sensors.csv

```diff
@@ -840,6 +840,8 @@ Nanometrics Inc.,Trillium Compact 120PH-2,3816,BHW,10,0,,0,0,0,0,0,0,2016-07-15T
 Nanometrics Inc.,Trillium Compact 120PH-2,3816,BHW,10,0,,0,0.7,0,0,0,0,2016-07-15T02:32:00Z,2020-07-15T01:01:00Z
 Nanometrics Inc.,Trillium Compact 120PH-2,3872,COVZ,10,335,,0,64,0,0,0,0,2016-04-07T03:30:00Z,9999-01-01T00:00:00Z
 Nanometrics Inc.,Trillium Compact 120PH-2,6637,WEL,13,0,,0,0,0,0,0,0,2021-03-30T03:00:00Z,2021-04-14T04:05:00Z
+Nanometrics Inc.,Trillium Compact 120PH-2,8743,WHSZ,10,0,,0,0,0,0,0,0,2021-11-03T22:57:00Z,9999-01-01T00:00:00Z
+Nanometrics Inc.,Trillium Compact 120PH-2,8785,WHSZ,11,0,,0,0,0,0,0,0,2021-11-03T22:57:00Z,9999-01-01T00:00:00Z
```

Comment:
I feel like this should have 293 in the depth column, I will update the azimuth later when the FDSN problems are resolved!

Reply:
`Nanometrics Inc.,Trillium Compact 120PH-2,8785,WHSZ,11,0,,0,293,0,0,0,0,2021-11-03T22:57:00Z,9999-01-01T00:00:00Z`

If I've counted the columns correctly

Reply:
Depth modified. Thanks @CBurton90